src/pages/SensorEditPage.js

```diff
@@ -2,8 +2,7 @@
 /* eslint-disable react/forbid-prop-types */
 /* eslint-disable react/jsx-wrap-multilines */
 import React from 'react';
-import { View, Text, StyleSheet } from 'react-native';
-import moment from 'moment';
+import { StyleSheet, ToastAndroid } from 'react-native';
```

Comment:
Main change in this component is extracting `FridgeHeader` into its own file and using it in here.